index.js

```diff
@@ -1,7 +1,68 @@
 // Iteration 1: Names and Input
-
-
+console.log("I'm ready!");
+const hacker1 = "Paulo";
+    console.log(`The driver's name is ${hacker1}.`);
+const hacker2 = "Gabriel";
+    console.log(`The navigator's name is ${hacker2}.`);
 // Iteration 2: Conditionals
-
-
+if (hacker1.length > hacker2.length) {
+    console.log("The driver has the longest name, it has " + hacker1.length);
```

Comment:
Here, you should've use string interpolation, like:
``The driver has the longest name, it has ${hacker1.length} characters.``